vpd-tool/include/vpd_tool.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <string>
+
+namespace vpd
+{
+/**
+ * @brief Class to support operations on VPD.
+ *
+ * The class provides API's to,
+ * Read keyword value from DBus/hardware.
+ * Update keyword value to DBus/hardware.
+ * Dump DBus object's critical information.
+ * Fix system VPD if any mismatch between DBus and hardware data.
+ * Reset specific system VPD keywords to its default value.
+ * Force VPD collection for hardware.
+ */
+class VpdTool
```

Comment:
why there is no constructor for class VpdTool?

Reply:
the class doesn't take any argument.